### PR TITLE
Truncate nicely

### PR DESCRIPTION
**Pull request by pikesley**
Currently, the Tweet is brutally truncated at 140 characters, which means it sometimes cuts off in the middle of a wo

This stops that from happening. Also removes the trailing comma, which can look a bit untidy

Also, specs :)

### CODE REVIEW

File: Guardfile [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    ]
+  end
+
+  # Rails config changes
+  watch(rails.spec_helper)     { rspec.spec_dir }
+  watch(rails.routes)          { "#{rspec.spec_dir}/routing" }
+  watch(rails.app_controller)  { "#{rspec.spec_dir}/controllers" }
+
+  # Capybara features specs
+  watch(rails.view_dirs)     { |m| rspec.spec.("features/#{m[1]}") }
```
Comment (parkr) [inline Guardfile]: Do we need these rails things?

File: lib/ebooks/twitter.rb [hunk 2]
```diff
@@ -18,9 +18,34 @@ def twitter_client
     end
 
     def tweet(tweet_text)
-      tweet_text = tweet_text.gsub('@', '')[0..139]
+      tweet_text = truncate tweet_text.gsub('@', '')
```
Comment (parkr) [inline lib/ebooks/twitter.rb]: I have a feeling this won't find `Ebooks.truncate` by itself. Am I incorrect? If so, why?

**Comment by parkr** [comment]
Thanks, @pikesley!

File: Guardfile (hunk 1 above)
Comment (pikesley) [inline Guardfile]: Nope, dinged 'em now

File: lib/ebooks/twitter.rb (hunk 2 above)
Comment (pikesley) [inline lib/ebooks/twitter.rb]: I have now fully-qualified the call. I really want to wrap some tests around all of this...

Comment (pikesley) [inline lib/ebooks/twitter.rb]: No wait, I'm still not sure about this

**Comment by pikesley** [comment]
I'm now happy with this, I think :)

**Comment by parkr** [comment]
Excellent, thank you!

**Pull request merged**